graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -403,10 +480,23 @@ def link_schema_elements(elements, inheritance_structure):
 #              explicit value for this property. Set to None if no default is given in the schema.
 PropertyDescriptor = namedtuple('PropertyDescriptor', ('type', 'default'))
 
-# A way to describe a schema's normalized inheritance structure.
+
+# A way to describe a schema's normalized inheritance structure:
 #   - superclass_sets: a dict, string -> set of strings, mapping each class to its
 #                      superclasses. The set of superclasses includes the class itself and
 #                      the transitive superclasses. For instance, if A is a superclass of B,
 #                      and B is a superclass of C, then C's superclass set is {'A', 'B', 'C'}.
 #   - subclass_sets: a dict, string -> set of strings, that is similarly defined.
 InheritanceStructure = namedtuple('InheritanceStructure', ('superclass_sets', 'subclass_sets'))
+
+
+# A way to describe an index:
+#   - name: string, the name of the index.
+#   - base_classname: string, the name of the class on which the index is defined.
+#   - fields: set of strings, the set of fields which the index encompasses.
+#   - unique: bool, indicating whether this index is unique.
+#   - ignore_nulls: bool, indicating if the index ignores null values.
+#   - ordered: bool, indicating whether this index is ordered.
```

Comment:
I'd keep `unique` and `ordered` adjacent to each other, and move `ignore_nulls` either above or below them, since that one is the unusual property here.